### PR TITLE
add service annotations

### PR DESCRIPTION
**Pull request by jungmann**
**What this PR does / why we need it**:

Add annotations to service. Eg use
```
service:
  annotations:
    service.beta.kubernetes.io/azure-load-balancer-internal: 'true'
```

to create internal loadbalancer on AKS.

### CODE REVIEW

File: stable/sonarqube/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 0.3.6
+version: 0.3.7
```
Comment (timstoop) [inline stable/sonarqube/Chart.yaml]: You're adding a new feature, so please bump the minor version.

File: stable/sonarqube/templates/service.yaml [hunk 2]
```diff
@@ -7,6 +7,10 @@ metadata:
     chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+{{- if .Values.service.annotations }}
```
Comment (timstoop) [inline stable/sonarqube/templates/service.yaml:10]: You'll have to add the key to values.yaml as well as providing some documentation about it in README.

**Review by timstoop — CHANGES_REQUESTED** [review]

**Comment by unguiculus** [comment]
Already implemented in #4060.

**Pull request closed without merging**